### PR TITLE
[sustainability] Add config and homepage for sustainability site

### PR DESCRIPTION
**Pull request by beets**
Adding v0 of a base template and homepage for the sustainability site.

### CODE REVIEW

**Comment by beets** [comment]
A screenshot of the homepage: 
image

**Comment by pradh** [comment]
Exciting! Go green! :)

File: server/configmodule.py [hunk 1]
```diff
@@ -37,15 +39,27 @@ class ProductionConfig(Config):
...
 
+class StagingSustainabilityConfig(StagingConfig):
+    SUSTAINABILITY = True
+
+
 class AutopushConfig(Config):
     pass
 
 
+class AutopushSustainabilityConfig(StagingConfig):
```
Comment (shifucun) [inline server/configmodule.py:59]: inherit AutopushConfig?

File: server/templates/sustainability/base.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+    <footer id="main-footer">
+      <div class="container">
+        {% if not is_hide_full_footer %}
+        <div class="row">
+          <section class="col-12 col-sm-6 col-md-4">
+            {# TRANSLATORS: The label for a collection of exploration tools. #}
+            <h6>{% trans %}Explore{% endtrans %}</h6>
+            {# TRANSLATORS: The name of a tool to browse statistics about a place. #}
+            <a href="{{ url_for('place.place') }}">{% trans %}Place Explorer{% endtrans %}</a>
+            {# TRANSLATORS: The name of a tool to browse the Data Commons knowledge graph. #}
```
Comment (shifucun) [inline server/templates/sustainability/base.html:106]: Are these translation comments needed here?

**Review by shifucun — APPROVED** [review]

**Comment by shifucun** [comment]
> Exciting! Go green! :)

Green is super nice here!

File: server/configmodule.py (hunk 1 above)
Comment (beets) [reply server/configmodule.py]: thanks for the catch!

File: server/templates/sustainability/base.html (hunk 2 above)
Comment (beets) [reply server/templates/sustainability/base.html]: thanks - not needed. removed.

**Pull request merged**